build/config-defuse.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/bash
+
+set -xe
+
+if [ "$#" -lt "1" ] ; then
+  echo "Please provide an installation path such as /opt/ICGC"
+  exit 1
+fi
+
+INST_PATH=$1
+INST_PATH=$(realpath $INST_PATH)
+
+if [ ! -d $INST_PATH/bin/defuse_install ]; then
+  echo "Could not find bin/defuse_install in $INST_PATH: Wrong installation path?"
+  exit 1
+fi
+
+if [ ! -d $INST_PATH/config ]; then
+  echo "Could not find folder 'config' in $INST_PATH: Wrong installation path?"
+  exit 1
+fi
+
+# how many spaces in the sed matching patter is important, do not change without close look.
+echo -e \
+"source_directory = $INST_PATH/bin/defuse_install\n"\
+"samtools_bin = $(which samtools)\n"\
```

Comment:
Do these actually need full path?  Do bare-word commands not work? You've extended `PATH` in the Docker sections so these should be available.